public/utils/contextMenu/monitors.ts

```diff
@@ -22,7 +22,7 @@ const getMonitors = async (params) => {
 
 const parseMonitor = (monitor) => {
   const state = monitor.monitor.enabled ? 'enabled' : 'disabled';
-  const latestAlert = monitor.latestAlert === "--" ? undefined : monitor.latestAlert;
```

Comment:
is there tests for `parseMonitor`? At least should be one added for this scenario

Reply:
Will follow up on tests and will create a tracking issue.

Reply:
#646